BREAKING.md

```diff
@@ -73,3 +73,155 @@ func createSQSAPI(endpoint string) (*sqs.Client, error) {
 ```
 
 > A more detailed documentation on migrating can be found [here](https://aws.github.io/aws-sdk-go-v2/docs/migrating).
+
+## v.0.74.0
```

Comment:
Very good and detailed approach on the migration, good job!

Since we want to keep the breaking changes in the form of a changelog, would you mind adding this change at the top of the file? So that `v0.74.0` would precede `v0.73.0`.